ConfigSpace/hyperparameters.pyx

```diff
@@ -1483,7 +1493,8 @@ cdef class CategoricalHyperparameter(Hyperparameter):
         repr_str.write("}")
         repr_str.write(", Default: ")
         repr_str.write(str(self.default_value))
-        if self.probabilities is not None:
+        # if the probability distribution is not uniform, write out the probabilities
+        if not np.all(self.probabilities == self.probabilities[0]):
```

Comment:
You removed the condition `if self.probabilities is not None:`, is `self.probabilities` guaranteed to exist now?

Reply:
Yes!